MonoGame.Framework/Platform/Utilities/FuncLoader.Desktop.cs

```diff
@@ -43,11 +43,16 @@ public static IntPtr LoadLibraryExt(string libname)
             // Try .NET Framework / mono locations
             if (CurrentPlatform.OS == OS.MacOSX)
             {
-                ret = LoadLibrary(Path.Combine(assemblyLocation, libname));
+                // Correct the working directory location for macOS .app bundles
+                // App's current working directory is actually `/`, so "./" won't work.
+                var currentDirectory = System.AppDomain.CurrentDomain.BaseDirectory;
+
+                // Look in MacOS for .app bundles
+                ret = LoadLibrary(Path.Combine(currentDirectory, libname));
```

Comment:
Now that I look at the entire file, this section is for .NET framework, your code needs to be added bellow.